Sources/UXKit/AppKit/UXGestures-AppKit.swift

```diff
@@ -26,6 +35,25 @@
       return self
     }
     
+    // This is how macOS handles Swipe gestures.
+    override func swipe(with event: NSEvent) {
+        let x : CGFloat = event.deltaX
+        let y : CGFloat = event.deltaY
+        var direction : SwipeDirection = .none
+        
+        if (x != 0) {
+            direction = (x > 0)  ? .left : .right
+        } else if (y != 0) {
+            direction = (y > 0)  ? .up : .down
+        }
+        
+        self.swipeGestureRecognized(inDirection: direction)
+    }
+    
+    // Override this if you want to receive swipe gestures on your NSView.
+    func swipeGestureRecognized(inDirection direction: SwipeDirection) {
```

Comment:
Wouldn't that func have to be `open` or `@objc` or sth to be actually subclassed?

The whole swipe thing doesn't seem to have any peer in UIKit? It is more like a convenience addition to just AppKit?